src/tests/import.rs

```diff
@@ -220,8 +220,8 @@ fn existing_peer_skip_import() {
 
 #[test]
 fn existing_peer_remove_unused() {
-    // (Pass) When pruning, we'll remove unused audits when unlocked, even if
-    // our peer hasn't changed. These audits will be preserved when not pruning.
+    // (Pass) When pruning, we'll remove unused audits (including violations) when unlocked, even
+    // if our peer hasn't changed. These audits will be preserved when not pruning.
```

Comment:
Could we re-wrap this comment?

Reply:
It is wrapped to 100 character lines. Want it shorter?

Reply:
One of these days they'll stabilize comment wrapping in rustfmt...